api/MediaQueryList.json

```
@@ -30,11 +30,11 @@
           },
           "safari": {
             "version_added": "5.1",
-            "notes": "In Safari, <code>MediaQueryList</code> is not based on <code>EventTarget</code>, so you must use <code>addListener()</code> and <code>remoteListener()</code> to observe media query lists. This is expected to change in macOS 11 Big Sur."
+            "notes": "Prior to Safari 14, <code>MediaQueryList</code> was not based on <code>EventTarget</code>, so you had to use <code>addListener()</code> and <code>removeListener()</code> to observe media query lists."
```

Can we roll these back to present tense? Safari 14 isn't even released yet, for starters. Like this:

```suggestion
            "notes": "Prior to Safari 14, <code>MediaQueryList</code> is based on <code>EventTarget</code>, so you must use <code>addListener()</code> and <code>removeListener()</code> to observe media query lists."
```